azure-pipelines.yml

```diff
@@ -32,21 +32,46 @@ steps:
     pmdRunAnalysis: true
     findBugsRunAnalysis: true
 
-# - task: Bash@3
-#   displayName: 'test example project with gradle'
-#   inputs:
-#     targetType: 'inline'
-#     script: 'rm -rf example-projects/example-project/target &&
-#       cd example-projects/example-project &&
-#       gradle analyze &&
-#       java -jar ../../core/frontend-stubs-testing/target/antenna-test-project-asserter.jar ExampleTestProject build'
-#     failOnStderr: true
+- task: Bash@3
+  displayName: 'test example project with maven'
+  inputs:
+    targetType: 'inline'
+    script: 'tmpdir=$(mktemp -d) &&
+      cp -r example-projects/example-project/ $tmpdir/example-project/ &&
+      rm -rf $tmpdir/example-project/target &&
+      pushd $tmpdir/example-project/ &&
+      mvn package &&
+      popd &&
+      java -jar core/frontend-stubs-testing/target/antenna-test-project-asserter.jar ExampleTestProject $tmpdir/example-project/target'
+
+- task: Bash@3
+  displayName: 'test example project with cli'
+  inputs:
+    targetType: 'inline'
+    script: 'tmpdir=$(mktemp -d) &&
+      cp -r example-projects/example-project/ $tmpdir/example-project/ &&
+      rm -rf $tmpdir/example-project/target &&
+      .travis/runCLI.sh $tmpdir/example-project &&
+      java -jar core/frontend-stubs-testing/target/antenna-test-project-asserter.jar ExampleTestProject $tmpdir/example-project/target'
 
-# - task: Bash@3
-#   displayName: 'test example project with cli'
-#   inputs:
-#     targetType: 'inline'
-#     script: 'rm -rf example-projects/example-project/target &&
-#       .travis/runCLI.sh example-projects/example-project &&
-#       java -jar core/frontend-stubs-testing/target/antenna-test-project-asserter.jar ExampleTestProject target'
-#     failOnStderr: true
+- task: Bash@3
+  displayName: 'test example project with gradle'
+  inputs:
+    targetType: 'inline'
+    script: 'tmpdir=$(mktemp -d) &&
+      cp -r example-projects/example-project/ $tmpdir/example-project/ &&
+      rm -rf $tmpdir/example-project/target &&
```

Comment:
Can't we call `./gradlew cleanAnalyze analyze` below instead of this (see [Automatic Clean Tasks](https://mrhaki.blogspot.com/2010/10/gradle-goodness-automatic-clean-tasks.html))?